formatTest/typeCheckedTests/expected_output/oo.re

```diff
@@ -30,28 +29,27 @@ let tmp = {
 /**
  * Comment on stackWithAttributes.
  */
```

Comment:
The input was:

```
/**
 * Comment on stackWithAttributes.
 */
class virtual stackWithAttributes 'a init =>
/* Before class */
{
```

Is the change intensional? In the new version, it looks like the comment is on the `as`, but in the original, it looked like it was on the entire class expression `{}`.